app/download/download.js

```diff
@@ -54,45 +54,16 @@ app.controller('DownloadCtrl', function($scope, $http, $uibModalInstance, $locat
 
 
   function downloadFile(fileName, data, strMimeType) {
-    var D = document;
-    var a = D.createElement('a');
     strMimeType = strMimeType || 'application/octet-stream;charset=utf-8';
-    var rawFile;
+    var exportLink = $('#quickGO-download-export-link');
```

Comment:
I am not sure what the purpose of `exportLink` is. It is also using jQuery which shouldn't be used there.